protocol/election-module/cannonfile.toml

```diff
@@ -0,0 +1,125 @@
+name = "synthetix-election-module"
```

Comment:
How does this account to the fact that there are 4 councils: spartan, treasury, grants, and ambassador

Reply:
you deploy one instance of this cannonfile for every council with a different salt.

If one council requires additional modules or etc. then an extension cannonfile should be created.

Reply:
Sweet